src/app/cli/src/tests/coda_restarts_and_txns_holy_grail.ml

```diff
@@ -29,26 +32,26 @@ let main n () =
   |> don't_wait_for ;
   (* RESTART NODES *)
   (* catchup *)
-  let%bind () = after (Time.Span.of_min 1.) in
+  let%bind () = after (Time.Span.of_min 2.) in
```

Comment:
since this expression is repeated, it might be useful to define:
```
  let wait_tm = Time.Span.of_min 2 in
```
and use that

Reply:
Sure.